### PR TITLE
Gpl 773 Sample Sheet v10 support

### LINKED ISSUE #509: GPL-773 Support SMRTLink Sample Sheet v10
Coupled with Traction UI [PR](https://github.com/sanger/traction-ui/pull/553)

**User story**
As developers we need to fix the errors caused by uploading sample_sheets from traction to the new smrtlink v10 as there is an upgrade planned for 14th December.

**Who are the primary contacts for this story**
Carol / Steve

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] upload a sample sheet to the smrtlink test instance without errors

**Tasks**
Service
- Add ‘Generate HiFi Reads
- Remove 'Generate CCS Data
- Remove  'Sequencing Mode' 
- Add ‘CCS Analysis Output - Include Kinetics Information

**Dependencies**
N/A

**Additional context**
You can access our test instance of smrtlink via this address:
-  https://trac-smrtlink-uat.psd.sanger.ac.uk:8243/sl/run-design

You can download sample sheets from any instance of traction from the pacbio run page:
However taking from production is advised as uat run data may not be complete
-  http://traction.psd.sanger.ac.uk/#/pacbio/runs

### PR DESCRIPTION
**Pull request by harrietc52**
Coupled with Traction UI [PR](https://github.com/sanger/traction-ui/pull/553)
Closes #509

### CODE REVIEW

**Comment by harrietc52** [comment]
TODO:
- check with Craig all existing Pacbio Runs have "In SMRT Link" inserted as Generate HiFi reads just after migration

Tasks:
- after migration populate all existing Pacbio runs  Generate HiFi  field with "In SMRT Link"

File: spec/models/pacbio/well_factory_spec.rb [hunk 1]
```diff
@@ -6,13 +6,13 @@
```
Comment (stevieing) [inline spec/models/pacbio/well_factory_spec.rb:6]: This is getting really big as well!

**Review by stevieing — APPROVED** [review]
All looks fine. Only thing I will say is that some of the spec files are massive and we might want to review them.

**Comment by harrietc52** [comment]
> All looks fine. Only thing I will say is that some of the spec files are massive and we might want to review them.

I've created an issue to log the clean up / refactoring we would like to do. https://github.com/sanger/traction-service/issues/515

**Pull request merged**